portfolio/src/main/webapp/style.css

```diff
@@ -81,3 +81,22 @@ button{
 button:hover{
     cursor: pointer;
 }
+
+form{
+    width: 50%;
+    min-width: fit-content;
+    border: solid red 3px;
+    background: linear-gradient(to bottom right, rgb(14, 158, 14), rgb(0, 139, 139), rgb(48, 48, 241));
+    padding: 1rem;
+    margin: 1rem;
```

Comment:
"Alphabetize declarations." This helps make this easier to maintain as it'll be easier to find a property.

https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order